packages/sources/genesis-volatility/src/adapter.ts

```diff
@@ -15,12 +16,13 @@ const daysConversion: Record<number, string> = {
   28: 'twentyEightDayIv',
 }
 
-const execute: ExecuteWithConfig<Config> = async (input, config) => {
-  const validator = new Validator(input, customParams)
+// TODO: Run tests with valid pro tier + API Key
```

Comment:
Make sure this TODO comment and the above are documented more that just here.
Otherwise they will just be lost.

Either a Github issue or a ticket on Clubhouse linking to these lines would be good.

Reply:
I created an issue here: https://github.com/smartcontractkit/external-adapters-js/issues/629